### DISCUSSION TITLE
[Feature Request] 提几个新组件

### DISCUSSION THREAD

**Discussion by liyuan1125** [youzan/vant#12313] [General]
### 这个功能解决了什么问题？

音频播放器：
例如：https://nutui.jd.com/h5/vue/4x/#/zh-CN/component/audio

视频播放器：
例如：https://nutui.jd.com/h5/vue/4x/#/zh-CN/component/video

拖拽：
例如：https://nutui.jd.com/h5/vue/4x/#/zh-CN/component/drag

引导：
例如：https://nutui.jd.com/h5/vue/4x/#/zh-CN/component/tour

表格
例如：https://nutui.jd.com/h5/vue/4x/#/zh-CN/component/table

结果页：
例如：https://mobile.ant.design/zh/components/result

以上组件在日常开发中能用到很多，但是vant目前还没有，如果用两三个ui框架的话 感觉又不伦不类，所以希望能考虑下是否可以添加

### 你期望的 API 是什么样子的？

示例有

**Comment by chenjiahan** [comment]
- 音频播放器/视频播放器：播放器属于重型组件，里面会涉及到视频格式的问题，我比较倾向于使用社区里单独的播放器 library，而不是集成到 Vant 里。
- 拖拽：这个组件和 Vant 里的 [FloatingBubble](https://vant-contrib.gitee.io/vant/#/zh-CN/floating-bubble) 有一些相似，我们应该不会再提供一个单独的拖拽组件了。
- 引导：引导也是比较复杂的，倾向于使用社区里单独的 library。
- 表格：table 有 issue 在跟进了：https://github.com/youzan/vant/issues/9180
- 结果页：结果页可以在 Vant 里提供，实现不难，主要问题是缺少相关的设计稿。。
